frontend/components/PermissionView.vue

```diff
@@ -18,6 +18,9 @@ defineEmits<{
 const perms = reactive(props.permissions);
 const api = useAPI();
 
+const availableLanguages = ref<string[]>([]);
+availableLanguages.value = (await api.getLanguages({environment: BmmEnvironment.Production})).Languages;
```

Comment:
It's usually best to do async things without await in setup, because calling `await` in setup makes the component an async component, and makes things complicated requiring to think about `<suspense>` etc.

e.g. with `then` or with https://nuxt.com/docs/api/composables/use-async-data
```ts
api.getLanguages({environment: BmmEnvironment.Production}).then((val) => { ... });
```